packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -239,6 +239,7 @@ public void init(
                     "default_web_client_id", "string", registrar.context().getPackageName());
         if (clientIdIdentifier != 0) {
           optionsBuilder.requestIdToken(registrar.context().getString(clientIdIdentifier));
+          optionsBuilder.requestServerAuthCode(registrar.context().getString(clientIdIdentifier));
```

Comment:
Is the serverClientId always static? We might want to have it be able to set through code as well. Same in iOS

Reply:
@cyanglaz 
Thank you for review and comment. I'm going to modify them.
About here, In my opinion I think serverClientId is static define, because it is not dynamic value.
but if you have idea or opinion, i give priority to them. please tell me :)

Reply:
returning serverAuthCode is using my flutter app. so I'm positive about support this modify.

Reply:
and if SERVER_CLIENT_ID value is null, return null.